### DISCUSSION TITLE
Request: PlantUML parser format

### DISCUSSION THREAD

**Discussion by sevenjay** [growilabs/growi#6324] [Ideas / Requests] [answered]
support PlantUML parser format like below
```
```plantuml
@startuml
[*] --> Initialed
Initialed --> Ready
```
```
```plantuml
[*] --> Initialed
Initialed --> Ready
```
keep @startuml or not is fine,
and other non-UML diagrams are also supported
```
```plantuml
@startjson
{
   "fruit":"Apple",
   "size":"Large",
   "color": ["Red", "Green"]
}
@endjson
```
There are many tools support this format https://notepm.jp/blog/6589
The context after only @startuml will be parser as markdown.  use ```plantuml will parser it as code

**Comment by yuki-takei** [comment]
related issue: https://github.com/weseek/growi/issues/6074

**Comment by yuki-takei** [comment, marked as answer]
## Update
https://github.com/weseek/growi/pull/6604

Please wait GROWI v6 which will be released the winter 2022.
